src/App.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import { v4 } from 'uuid';
+import './App.css';
+
+class App extends Component {
+  state = {
```

Comment:
Why not store this initial state in a separate js file, or a json file?

Reply:
I've made the change - I had to do it in a .js file due to using the uuid module for the todo id generation, I'm not sure there was a way to get around this by using a json file.